integrations/clion.rst

```diff
@@ -13,17 +13,199 @@ Just include the ``conanbuildinfo.cmake`` this way:
 
 .. code-block:: cmake
 
-   if(EXISTS ${CMAKE_CURRENT_SOURCE_DIR}/build/conanbuildinfo.cmake) #Clion, with conanbuildinfo.cmake in build folder
-       include(${CMAKE_CURRENT_SOURCE_DIR}/build/conanbuildinfo.cmake)
+   if(EXISTS ${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+       include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+       conan_basic_setup()
    else()
-       include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake) # Not CLion
+       message(WARNING "Skipping Conan")
    endif()
-   
-   
-.. note::
-   In the example above we generated the ``conanbuildinfo.cmake`` in the **build** folder running: "conan install --file ../conanfile.txt" in that folder.
-   You can use other directory or project root if you want, just change the route in CMakeLists include of ``conanbuildinfo.cmake`` file.
 
+If the ``conanbuildinfo.cmake`` file is not found, it will print a warning message in the ``Messages`` console of your Clion IDE.
 
 
-.. |clion_logo| image:: ../images/icon_CLion.png
+Using packages in a CLion project
+=================================
+
+Let see an example of how to use Conan requirements to a CLion project.
+
+1. Create a new CLion project
+
+|wizard_new|
+
+2. Edit the ``CMakeLists.txt`` file and add the following lines:
+
+.. code-block:: cmake
+
+   if(EXISTS ${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+       include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+       conan_basic_setup()
+   else()
+       message(WARNING "Skipping Conan")
```

Comment:
I'd say to use a better message, like "conanbuildinfo.cmake doesn't exist, you have to run ``conan install``", or something similar.